src/environments/environment.prod.ts

```diff
@@ -1,7 +1,7 @@
 export const environment = {
   production: true,
   uploadImageHostname: "bitclout.com",
-  jumioEndpointHostname: "bitclout.com",
+  jumioEndpointHostname: "localhost:18001",
```

Comment:
Is this correct? It seems like "environment.prod.ts" should point to bitclout.com for the jumio endpoint host

Reply:
this is just for testing purposes - we'll flip this back once we're done testing.